### PR TITLE
Manage sysconfig

### PR DESCRIPTION
**Pull request by ghoneycutt**

### CODE REVIEW

**Comment by ghoneycutt** [comment]
@Phil-Friderici Could you please help out with the gem dependencies? Having a hard time getting this to pass on all the platforms.

File: Gemfile [hunk 1]
```diff
@@ -23,10 +23,13 @@ gem 'puppet-lint-variable_contains_upcase'
 
 gem 'rspec',              '~> 2.0'   if RUBY_VERSION >= '1.8.7' && RUBY_VERSION < '1.9'
 gem 'rake',               '~> 10.0'  if RUBY_VERSION >= '1.8.7' && RUBY_VERSION < '1.9'
+gem 'public_suffix',      '~> 2.0'   if RUBY_VERSION >= '2.0.0' && RUBY_VERSION < '2.1'
```
Comment (jeffmccune) [inline Gemfile]: With rvm `2.1.0preview1` I ended up using `gem 'public_suffix',      '~> 2.0'   if RUBY_VERSION < '2.1.1'`

**Comment by ghoneycutt** [comment]
@jeffmccune Thanks!

Released in v2.1.0

File: Gemfile (hunk 1 above)
Comment (Phil-Friderici) [reply Gemfile]: The public_suffix gem came in with an update of metadata-json-lint. I fixed it by pinning metadata-json-lint to 1.1.0 for Rubies < 2.1:
```
gem 'metadata-json-lint',     '0.0.11'   if RUBY_VERSION >= '1.8.7' && RUBY_VERSION < '1.9'
gem 'metadata-json-lint',     '1.1.0'    if RUBY_VERSION >= '1.9' && RUBY_VERSION < '2.1'
gem 'metadata-json-lint' if RUBY_VERSION >= '2.1'
```

https://github.com/gillarkod/puppet-vsftpd/blob/1.0.5/Gemfile#L27-L29

**Pull request merged**